includes.mk

```diff
@@ -0,0 +1,12 @@
+COMPONENT ?= workflow
```

Comment:
Gotcha.  In that case, since we are currently only building/pushing all of workflow as a Docker image, with no immediate plans to build/push a Docker image of the client (right?), and since COMPONENT and VERSION wish not to be shared, perhaps the `includes.mk` is a bit premature at this point?